inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/conversion.cpp

```diff
@@ -18,24 +18,27 @@ const std::vector<ngraph::helpers::ConversionTypes> conversionOpTypes = {
 const std::vector<std::vector<size_t>> inShape = {{1, 2, 3, 4}};
 
 const std::vector<InferenceEngine::Precision> netPrecisions = {
-    // Ticket: 59594
-    // InferenceEngine::Precision::I4,
+    InferenceEngine::Precision::U8,
     InferenceEngine::Precision::I8,
+    InferenceEngine::Precision::U16,
     InferenceEngine::Precision::I16,
+    InferenceEngine::Precision::U32,
     InferenceEngine::Precision::I32,
-    InferenceEngine::Precision::I64,
-    // Ticket: 59594
-    // InferenceEngine::Precision::BIN,
-    // InferenceEngine::Precision::BOOL,
-    // InferenceEngine::Precision::U4,
-    InferenceEngine::Precision::U8,
-    InferenceEngine::Precision::U16,
-    // Ticket: 59594
-    // InferenceEngine::Precision::U32,
     InferenceEngine::Precision::U64,
+    InferenceEngine::Precision::I64,
     InferenceEngine::Precision::BF16,
     InferenceEngine::Precision::FP16,
-    InferenceEngine::Precision::FP32};
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP64,
+    InferenceEngine::Precision::BOOL,
+    // Ticket: 59594
+    // InferenceEngine::Precision::MIXED,
```

Comment:
Maybe will be better to use skip_tests_config and uncomment these lines?

Reply:
done